src/main/kotlin/no/nav/syfo/cronjob/SuspensjonCronjob.kt

```diff
@@ -0,0 +1,64 @@
+package no.nav.syfo.cronjob
+
+import net.logstash.logback.argument.StructuredArguments
+import no.nav.syfo.behandler.BehandlerService
+import no.nav.syfo.client.btsys.LegeSuspensjonClient
+import org.slf4j.LoggerFactory
+import java.time.Duration
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.time.LocalTime
+
+class SuspensjonCronjob(
+    val behandlerService: BehandlerService,
+    val legeSuspensjonClient: LegeSuspensjonClient,
+) : DialogmeldingCronjob {
+    private val runAtHour = 11
```

Comment:
Hvorfor klokken 11? 🤔

Reply:
Bare for testing nå. Skal endre til 5 før merge.